assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -115,6 +115,14 @@ che.workspace.agent.dev.ping_timeout_error_msg=Timeout. The Che server is unable
 che.agent.dev.max_start_time_ms=120000
 che.agent.dev.ping_delay_ms=2000
 
+# Idle Timeout
+#     The system will suspend the workspace and snapshot it if the end user is idle for
+#     this length of time. Idleness is determined by the length of time that a user has
+#     not interacted with the workspace, meaning that one of our agents has not received
+#     instructions. Leaving a browser window open counts as idleness time.
+che.machine.ws.agent.inactive.stop.timeout.ms=3600000
+stop.workspace.scheduler.period=60
```

Comment:
Please prepend property name with `che.` and add measurement unit as a suffix of the property.